### DISCUSSION TITLE
How preview a HTML file?

### DISCUSSION THREAD

**Discussion by kisvegabor** [community/community#39473] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Let's say I have this `index.html` file in GitHub Codespaces:
```html
<h1>Hello world</h1>
Test text
```

How can I preview the HTML file in GitHub Codespaces? Preferably on a VS code tab.

**Comment by kisvegabor** [comment]
Hey,

I've just tried it out but there is no preview option for me neither if I right-click on the file:
![image](https://github.com/community/community/assets/7599318/d281a72c-176b-4be1-a2ae-6493130dfe46)

nor I right-click in the editor window:
![image](https://github.com/community/community/assets/7599318/55b6c12a-3889-48bd-aca6-52680b15f978)

**Comment by GijsSi** [comment, marked as answer]
Hey @kisvegabor If you are using Virtual Studio Code, what I always do is download the Live Preview plugin and then a button will appear in the right corner. Which you can click and then it will show changes directly on the page. (It opens up a web browser page which is your html file)

Let me know if it helped :)

**Comment by kisvegabor** [comment]
Confirmed! The Live preview plugin really does the trick! Thank you!
